### PR TITLE
Feature/badges

### PR DESCRIPTION
**Pull request by bhanuprakashj**
image
To check the output please refer readme file

### CODE REVIEW

File: src/css/abstracts/variables.css [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	--beauty-bush: #efb5ba;
 	--saddle-brown: #564000;
 	--double-colonial-white: #efdfab;
+	--san-felix: #096915;
```
Comment (FarhanMobashir) [inline src/css/abstracts/variables.css:16]: Don't keep the color name as css variable as it doesn't make sense. Keep the shades of colours instead like blue 100 , blue 200 and similar as described in the Design system guide on lms

File: src/css/components/badge.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+.badge-container {
+	display: inline-block;
+	position: relative;
+	font-size: 2rem;
+}
+
+.badge-sm {
```
Comment (FarhanMobashir) [inline src/css/components/badge.css]: Make a class of badge and give the basic styling and now add badge sm class after that by reducing the size

**Review by FarhanMobashir — COMMENTED** [review]
Addes some comments

File: src/css/abstracts/variables.css [hunk 3]
```diff
@@ -12,6 +12,7 @@
 	--beauty-bush: #efb5ba;
```
Comment (lannisterrr) [inline src/css/abstracts/variables.css:12]: use primary and secondary color class name for a good design system.

**Review by lannisterrr — COMMENTED** [review]
Other than points mentioned above , the code seems fine.

File: src/css/abstracts/variables.css (hunk 3 above)
Comment (bhanuprakashj) [reply src/css/abstracts/variables.css:12]: Thanks Siddant, Changed the variables accordingly will add the primary/secondary colors to the website while working on documentation site

File: src/css/abstracts/variables.css (hunk 1 above)
Comment (bhanuprakashj) [reply src/css/abstracts/variables.css:16]: Thanks Farhan for the suggestion, changed it accordingly

File: src/css/components/badge.css (hunk 2 above)
Comment (bhanuprakashj) [reply src/css/components/badge.css]: Addressed it!, thanks!!

**Pull request merged**